bundler/spec/install/gems/compact_index_spec.rb

```diff
@@ -893,15 +893,15 @@ def start
       expect(exitstatus).to eq(19)
       expect(err).
         to eq <<~E.strip
-          Bundler cannot continue installing rack (1.0.0).
+          Bundler cannot continue installing rack-1.0.0.
           The checksum for the downloaded `rack-1.0.0.gem` does not match the known checksum for the gem.
           This means the contents of the downloaded gem is different from what was uploaded to the server or first used by your teammates, and could be a potential security issue.
 
           To resolve this issue:
           1. delete the downloaded gem located at: `#{gem_path}`
           2. run `bundle install`
 
-          If you are sure that the new checksum is correct, you can remove the `rack (1.0.0)` entry under the lockfile `CHECKSUMS` section and rerun `bundle install`.
+          If you are sure that the new checksum is correct, you can remove the `rack-1.0.0` entry under the lockfile `CHECKSUMS` section and rerun `bundle install`.
```

Comment:
this shouldn't have changed, because `rack (1.0.0)` is the format used in the lock file

Reply:
ok, good point. Let's me see if I can seed the error message correctly.